SharedPackages/AIChat/Sources/AIChat/iOS/PublicAPI/AIChatViewController.swift

```diff
@@ -48,9 +48,9 @@ public final class AIChatViewController: UIViewController {
     public weak var delegate: AIChatViewControllerDelegate?
     private let chatModel: AIChatViewModeling
     private var webViewController: AIChatWebViewController?
-    private let chatInputHostingController: UIHostingController<AnyView>?
-    private var chatInputHeightConstraint: NSLayoutConstraint?
-    private var chatInputBottomConstraint: NSLayoutConstraint?
+    private var chatInputBoxViewController: UIViewController?
```

Comment:
Changed the dependency to a generic UIViewController so it’s easier to share the component, since it needs to play nice with our current UIKit UIs